packages/manager/apps/pci-workflow/src/api/hooks/order.ts

```diff
@@ -1,44 +1,57 @@
-import { useQuery } from '@tanstack/react-query';
-import { getCatalogQuery, useProject } from '@ovh-ux/manager-pci-common';
+import { useQueries } from '@tanstack/react-query';
+import {
+  getCatalogQuery,
+  getProductAvailabilityQuery,
+} from '@ovh-ux/manager-pci-common';
 
-export const useProjectAddons = (projectId: string, ovhSubsidiary: string) => {
-  const { data: project } = useProject(projectId);
-  return useQuery({
-    ...getCatalogQuery(ovhSubsidiary),
-    enabled: !!project && !!ovhSubsidiary,
-    select: (catalog) => {
-      const projectPlan = catalog.plans.find(
-        ({ planCode }) => planCode === project.planCode,
-      );
-      return [
-        ...new Set(projectPlan.addonFamilies.map((f) => f.addons).flat()),
-      ].map((addon) =>
-        catalog.addons.find(({ planCode }) => planCode === addon),
+export const useProjectAddons = (
+  projectId: string,
+  ovhSubsidiary: string,
+  instanceRegion: string,
+) =>
+  useQueries({
+    queries: [
+      getCatalogQuery(ovhSubsidiary),
+      getProductAvailabilityQuery(projectId, ovhSubsidiary),
+    ],
+
+    combine: ([
+      { data: catalog, isFetching: isCatalogFetchibng },
+      { data: plans, isFetching: isPlansFetching },
+    ]) => {
+      const plan = plans?.plans?.find(
+        ({ code, regions }) =>
+          code.startsWith('snapshot.consumption') &&
+          regions.find(({ name }) => name === instanceRegion),
       );
+      const addons = [
+        catalog?.addons.find(({ planCode }) => planCode === plan.code),
```

Comment:
I think `plan` can be undefined
```suggestion
        const addons = [
         catalog?.addons.find(({ planCode }) => planCode === plan?.code)
         ];
```